src/components/interface/dialog/DialogInput.jsx

```diff
@@ -41,7 +43,8 @@ DialogInput.propTypes = {
   name: PropTypes.string.isRequired,
   fullWidth: PropTypes.bool,
   helperText: PropTypes.string,
-  label: PropTypes.string,
+  InputProps: PropTypes.node,
```

Comment:
I don't believe this is a node, rather since `InputProps` expects an object, it should be `PropTypes.shape`. 

Also, are there any assurances we need on the shape of the `shape` object?

Ref: https://reactjs.org/docs/typechecking-with-proptypes.html

Take a look at `optionalObjectWithShape`